src/patternfly/utilities/Sizing/examples/sizing-button-example.hbs

```diff
@@ -0,0 +1,3 @@
+{{#> button button--modifier="pf-m-primary pf-m-block pf-u-w-initial-on-md"}}
```

Comment:
this button example has a dashed border, probably due to some workspace scss

Reply:
I added some CSS to the workspace to specifically target this one example. I suppose this button demo could be removed too, although it's a good illustration of its value.